packages/storefront-lwc/src/modules/commerce/lineItemTotal/lineItemTotal.html

```diff
@@ -6,12 +6,12 @@
 -->
 
 <template>
-    <div class="pricing line-item-total-discout-amount">
-        ${itemDiscountTotal}
+    <div class="pricing line-item-total-discount-amount">
+        ${discountTotal}
```

Comment:
this change was needed because product card prices were not being updated when the data passed in was new/changed.